charts/ups-broker/templates/broker-service.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
```

Comment:
Concern: does this result in an unpredictable name for this service?  It's important that the name be predictable so we can use a predictable DNS name in the walkthrough resource for the broker.